pkgs/os/default.nix

```diff
@@ -0,0 +1,30 @@
+{ pkgs ? import <nixpkgs> {}
+, toolbox
+}:
```

Comment:
`s/pkgs ? import <nixpkgs> {}/pkgs/`
Remove `toolbox`
You can pass `openstackclient vault curl jq pkgs.sedutil`